src/services/apimap-sender.js

```diff
@@ -1,10 +1,12 @@
 const request = require('superagent');
-const ServiceUrlGetter = require('./service-url-getter');
+const context = require('../context');
```

Comment:
[CLEANING] You might want to do this instead: 
`const { inject } = require('../context');`